nixos/modules/services/network-filesystems/ceph.nix

```diff
@@ -230,8 +230,15 @@ in
       };
       package = lib.mkPackageOption pkgs "ceph" { };
       extraConfig = lib.mkOption {
-        type = with lib.types; attrsOf str;
+        type = with lib.types; attrsOf (attrsOf str);
```

Comment:
Can we change this to submodule? otherwise it merged really bad in the module system